sdk/core/azure-core-opentelemetry/src/opentelemetry.cpp

```diff
@@ -0,0 +1,179 @@
+
```

Comment:
file header

Reply:
Surprised this isn't in the CI pipeline.